internal/commands/permission.go

```diff
@@ -100,15 +100,15 @@ func RegisterPermissionCmd(rootCmd *cobra.Command) *cobra.Command {
 
 	lookupResourcesCmd := &cobra.Command{
 		Use:               "lookup-resources <type> <permission> <subject:id>",
-		Short:             "Enumerates the resources of a given type for which the subject has permission",
+		Short:             "Enumerates the resources of a given type for which a subject has permission",
```

Comment:
Hmm... I read this as being in the context of the `Use` field, in which case there is a subject in question. Not gonna die on that hill tho.